modules/core/src/lib/deck.js

```diff
@@ -254,6 +271,11 @@ export default class Deck {
       this.viewState = props.initialViewState;
     }
 
+    // UNSAFE/experimental props
+    if (props._typedArrayManagerProps) {
```

Comment:
I don’t think it’s necessary to support changing this prop after initialization. It’s unclear what purpose this will serve - e.g. existing attributes wouldn’t be restored if you turn optimization off.

Reply:
Agreed that it could be acceptable to limit this to be an initialization-only prop, however `TypedArrayManager` is a singleton, so we don't have access to its initialization, which is why I opted for this approach. 

Any suggestions?

Reply:
Just set it once in Deck's constructor.